src/orders/domain/order.domain.ts

```diff
@@ -0,0 +1,49 @@
+import Customer from '../../customers/domain/customer.domain';
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity()
+export default class Order {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @ManyToOne(() => Customer)
+  @JoinColumn({ name: 'customer_id' })
+  customer: Customer;
+
+  @Column({ nullable: false })
+  createdAt: string;
+
+  @Column({ nullable: false })
+  country: string;
+
+  @Column({ nullable: false })
+  city: string;
+
+  @Column({ nullable: false })
+  county: string;
+
+  @Column({ nullable: false })
+  streetAddress: string;
+
```

Comment:
Adding the relation to `OrderDetail` might help you in your implementation (for "create order" operation).

Reply:
Added One-To-Many relation to Order => OrderDetail